Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM php:5.6
+
+RUN echo "date.timezone = UTC" > /usr/local/etc/php/conf.d/date.ini
```

Comment:
You should really add this:
```
apt-get update && \
apt-get install -y libicu-dev g++ && \
docker-php-ext-install intl mbstring opcache && \
apt-get clean && rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/* && \
```

It's for the `app/check.php` and also to add `opcache`